### PR TITLE
Add Python script/instructions for writing multiple Items to a Firestore User

### PR DESCRIPTION
**Pull request by owencooke**
See the README.md in this commit for context.

### CODE REVIEW

**Review by MatthewNeufeld — APPROVED** [review]
Works like a charm and everything looks clean. Pretty cool that we have this. LGTM.

File: test_data/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+Run the script with the following command:
+
+```bash
+python3 fb_batch_write.py <username> <path/to/data.json> <path/to/service/account/key.json>
+```
+
+where:
+
```
Comment (Sami-Jagirdar) [inline test_data/README.md:33]: Just a nitpick, we should probably mention in the readme what the format of the JSON file should be (like description, date and cost are mandatory fields). 
Also, do we team members have access to the account key for the project firebase? If yes, where can I access it?

File: test_data/fb_batch_write.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import sys
+import json
+from datetime import datetime
+from firebase_admin import credentials, initialize_app
+from google.cloud import firestore
+
+
+def import_data_to_firestore(db, test_user, data):
+    collection_path = f"user/{test_user}/item"
```
Comment (Sami-Jagirdar) [inline test_data/fb_batch_write.py:9]: I assume that the username must already exist in the firebase for batch write to work right?

**Review by ldbonkowski — APPROVED** [review]
LGTM!

File: test_data/fb_batch_write.py (hunk 2 above)
Comment (owencooke) [reply test_data/fb_batch_write.py:9]: Added a function that creates the user document if it does not exist, allowing this script to be ran for any username 💪

File: test_data/README.md (hunk 1 above)
Comment (owencooke) [reply test_data/README.md:57]: Updated the README accordingly. I will share the account key in our Discord server!

**Comment by owencooke** [comment]
@Sami-Jagirdar updated the PR with some fixes to address ur comments!

**Comment by Sami-Jagirdar** [comment]
@owencooke Awesome stuff! LGTM too.

**Pull request merged**